cinema/views.py

```diff
@@ -0,0 +1,44 @@
+from django.shortcuts import get_object_or_404
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+
+from cinema.models import Movie
+from cinema.serializers import MovieSerializer
+
+
+# Create your views here.
+@api_view(["GET", "POST"])
+def movie_list(request):
+    if request.method == "GET":
+        movie = Movie.objects.all()
+        serializers = MovieSerializer(movie, many=True)
+        return Response(serializers.data, status=status.HTTP_200_OK)
+
+    if request.method == "POST":
+        serializers = MovieSerializer(data=request.data)
+
+        if serializers.is_valid(raise_exception=True):
+            serializers.save()
+            return Response(serializers.data, status=status.HTTP_201_CREATED)
+
+        return Response(serializers.data, status=status.HTTP_400_BAD_REQUEST)

```

Comment:
```suggestion

```

No need of this line, because `raise_exception=True` in `serializer.is_valid()` will handle it